registry-library/library/library.go

```diff
@@ -329,7 +330,7 @@ func PullStackByMediaTypesFromRegistry(registry string, stack string, allowedMed
 	// Decompress archive.tar
 	archivePath := filepath.Join(destDir, "archive.tar")
 	if _, err := os.Stat(archivePath); err == nil {
-		err := decompress(destDir, archivePath)
+		err := decompress(destDir, archivePath, []string{"OWNERS"})
```

Comment:
Could you declare the exclude list in this block? https://github.com/devfile/registry-support/blob/c45fcd671516034dd299cfce2e7db718fefa3602/registry-library/library/library.go#L54